pkg/flowcli/services/transactions.go

```diff
@@ -147,29 +197,50 @@ func (t *Transactions) send(
 	return tx, res, err
 }
 
-// GetStatus of transaction
-func (t *Transactions) GetStatus(
-	transactionID string,
-	waitSeal bool,
+// SendForAddressWithCode send transaction for address and private key specified with code
+func (t *Transactions) SendForAddressWithCode(
+	code []byte,
+	signerAddress string,
+	signerPrivateKey string,
```

Comment:
Why pass the private key as a string? How would this work with the KMS signer?

Reply:
This implementation is needed by the langauge server and is not supposed to be used with KMS. I will revisit this with Max if there is a better way to implement this but didn't want to block him.